vnext/ReactUWP/Views/Image/ReactImage.cpp

```diff
@@ -90,10 +92,14 @@ namespace react {
             m_onLoadEndEvent(*this, false);
           }
         }
+        else if (inlineData)
+        {
+          memoryStream = co_await GetImageInlineDataAsync(source);
```

Comment:
>memoryStream = co_await GetImageInlineDataAsync(source); [](start = 10, length = 56)

Same as above to fire loaderror:
if (!memoryStream) { m_onLoadEndEvent(*this, false); }